assets/src/components/TableAssignment.js

```diff
@@ -89,13 +89,21 @@ function CustomAssignmentTable (props) {
                   let displayProp = true
                   let displayBorder = true
                   if (data[i - 1]) {
-                    if (data[i][0] === data[i - 1][0] && j < 2) {
+                    // first item in the row the logic handles display of week property
+                    if (j == 0 && data[i][0] === data[i - 1][0]) {
+                      displayProp = false
+                    }
+                    // second item in the row the logic handles display of Due date property
+                    if(j == 1 && data[i][1] === data[i-1][1]){
```

Comment:
@pushyamig could you pull my latest change? I fixed the styling issues by removing border-bottom and props of certain cells if the previous or next row had the same week. It works on my test course, please check with yours!
I changed few things in logic to take into account due date 

 some week may have more than one due date Adding logic to consider due date as well